<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Chivo&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>251</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="9" column="0" colspan="3">
      <widget class="QCheckBox" name="forbiddenCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Include forbidden reflections</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="cifnameLine">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Conventional XRD experiment</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="bLine">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>gamma (º)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="aLine">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>b (Å)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>a (Å)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="betaLine"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="cLine"/>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="gammaLine"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>c (Å)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>beta (º)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="alphaLine"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>alpha (º)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="updateButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 210, 255);</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="loadcifButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 210, 255);</string>
       </property>
       <property name="text">
        <string>Load .cif file</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QTableWidget" name="reflectionTable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <column>
        <property name="text">
         <string>Reflection</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Intensity</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QPushButton" name="fetchreflectionsButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 210, 255);</string>
       </property>
       <property name="text">
        <string>Fetch active reflections</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>maximum h, k l:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="maxhklLine">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="generatecifButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 210, 210);</string>
       </property>
       <property name="text">
        <string>Generate .cif</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>676</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="3">
      <widget class="QLineEdit" name="e_stepLine">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QTableWidget" name="edgeTable">
       <property name="maximumSize">
        <size>
         <width>129</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <column>
        <property name="text">
         <string>Atom</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="e_startLine">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QPushButton" name="launchButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 210, 255);</string>
       </property>
       <property name="text">
        <string>Launch simulation</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" rowspan="3" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Percentage for variation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">color: rgb(126, 126, 126)</string>
       </property>
       <property name="text">
        <string>step</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Edge for simulation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Atomic positions</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">color: rgb(126, 126, 126)</string>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="7">
      <widget class="QTableWidget" name="positionTable">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <column>
        <property name="text">
         <string>Atom</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>x</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>z</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Occupation</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QPushButton" name="sensitivityButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(103, 142, 172);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Sensitivity versus 
Intensity</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" rowspan="3">
      <widget class="QLineEdit" name="percentLine">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">color: rgb(126, 126, 126)</string>
       </property>
       <property name="text">
        <string>start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="e_endLine">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="13" column="3" rowspan="6" colspan="4">
      <layout class="QVBoxLayout" name="graphicLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Energy for simulations (eV):</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="5" colspan="2">
      <widget class="QCheckBox" name="coupleCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Couple refinement parameters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="3" rowspan="3" colspan="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QCheckBox" name="normalisedCheckbox">
       <property name="text">
        <string>Normalised sensitivity</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLineEdit" name="nsimLine">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Number of simulations:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
